hydra/conf/__init__.py

```diff
@@ -98,13 +98,18 @@ class HydraConf:
             {"hydra_help": "default"},
             {"hydra_logging": "default"},
             {"job_logging": "default"},
+            {"callbacks": "default"},
```

Comment:
I think if you use None here you won't need an empty file:
```suggestion
            {"callbacks": None},
```

This is equivalent to using null in the defaults list in yaml.
(documented [here](https://hydra.cc/docs/next/advanced/defaults_list)).